packages/server/postgres/migrations/1647457468652_addAzureDevOpsAuthTable.ts

```diff
@@ -0,0 +1,33 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+
+  await client.query(`
+    CREATE TABLE IF NOT EXISTS "AzureDevOpsAuth" (
```

Comment:
-2 This data should go into the "TeamMemberIntegrationAuth" table. We did not finalize where we're storing `instanceIds` but these should probably also go into "TeamMemberIntegrationAuth".

Reply:
tagging @wdfox

Reply:
This has been updated.  Tagging @wdfox